.github/workflows/deploy.yml

```
@@ -0,0 +1,49 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Set up Node.js
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)